sentinel-core/src/main/java/com/alibaba/csp/sentinel/node/metric/MetricTimerListener.java

```diff
@@ -40,9 +41,13 @@
     public void run() {
         Map<Long, List<MetricNode>> maps = new TreeMap<>();
         for (Entry<ResourceWrapper, ClusterNode> e : ClusterBuilderSlot.getClusterNodeMap().entrySet()) {
-            ClusterNode node = e.getValue();
-            Map<Long, MetricNode> metrics = node.metrics();
-            aggregate(maps, metrics, node);
+            if (SentinelConfig.shouldFilterMetric() && RuleProvider.isNotInRules(e.getValue().getName())) {
+                RecordLog.debug("[MetricTimerListener] Metric filter by config", e.getValue());
+            } else {
```

Comment:
We may only log the resource name, since `e.getValue()` is instance of `ClusterNode` type, which has not override `toString()` method, and add a space after `by config"` is better.

Reply:
OK, I have fixed it.

Reply:
@furaul It seems not be fixed, `e.getValue()` -> `e.getValue().getName()`

Reply:
> @furaul It seems not be fixed, `e.getValue()` -> `e.getValue().getName()`
My fault, it is fixed this time.